docs/organizations/managing-security-and-risk.md

```diff
@@ -27,86 +27,89 @@ The main area of the dashboard includes five panels:
+-   You [ignore the associated issue](../repositories/issues.md#ignoring-and-managing-issues)
+-   You [disable the tool](../repositories-configure/configuring-code-patterns.md) that found the associated issue
 
-    !!! note
-        Use a Jira account with admin permissions when authorizing Codacy. This lets Codacy access all issues, since the integration inherits the permissions of the authorizing account.
+!!! important
+    Deleting a repository deletes all open items belonging to that repository.
 
-    ### Uninstalling the Jira Cloud integration
+### How Codacy manages items detected on Jira {: id="opening-and-closing-jira-items"}
 
-    To uninstall the Jira integration:
+!!! note
+    -   For Codacy to detect Jira issues, you must [integrate Jira with Security and risk management](./integrations/jira-integration.md).
+    -   Codacy retrieves updates from Jira once a day. If an issue is opened and closed on the same day, Codacy may not detect it.
+    -   To make sure that Codacy detects Jira issues correctly, assign the **security** label when creating the issue or immediately after.
 
-    1.  Open your organization **Security and Risk** page and click the button **Configure** to open the configuration page.
+Codacy opens a new item when it detects a new Jira issue with a **security** label (case-insensitive).
 
-    1.  Find the Jira entry under the Integrations list and click the **Options** button (identified by three dots), then select **Uninstall** and confirm.
+Codacy closes an item when it detects that the associated Jira issue is marked as Closed.
 
-## Managing access to security and risk management
+## Managing access to Security and risk management
 
-To better track and address security issues, organization admins can extend access to security and risk management by assigning the Security Manager role to organization members.
+To better track and address security issues, organization admins can extend access to Security and risk management by assigning the Security Manager role to organization members.
 
 The Security Manager role is independent of the Git provider role of an organization member and provides only the permissions necessary to monitor security issues, following the principle of least privilege:
 
 | Permission                                                  | Organization Admin | Security Manager           |
 |-------------------------------------------------------------|--------------------|----------------------------|
-| Access security and risk management                         | Yes                | Yes                        |
+| Access Security and risk management                         | Yes                | Yes                        |
 | Access associated issues (Codacy repositories, Jira issues) | Platform-dependent | Keeps original permissions |
 | Manage integrations                                         | Yes                | Yes                        |
```

Comment:
Although we never "officially" refer to Slack as a SRM integration in the docs 🤔 

Should we start thinking about creating a new doc page only for these dynamic permissions? IIRC these will eventually outgrow SRM

Reply:
I'm good if you think it's clear in this context that "Manage integrations" only refers to the Jira integration (i.e. visible in the SRM configuration page) 👍

Reply:
We may still show a link to the Slack integration on the SRM config page (see [this thread](https://www.figma.com/file/Zw8KLyYC3Erg3ccYyb9lsz?node-id=2550:22109&mode=design#552271082)). That may confuse users, since it associates Slack with SRM...

I'll see how it looks once implemented and clarify if needed.

Reply:
I suggest going with "Manage the Jira integration".

It will still be unclear what Slack is doing there. I don't want to block this though and will review the entirety of the page in a follow-up task ([DOCS-584](https://codacy.atlassian.net/browse/DOCS-584)).